pkg/utils/common.go

```diff
@@ -559,3 +559,11 @@ func Command(cmdLine string, params ...string) *exec.Cmd {
 	c.Env = os.Environ()
 	return c
 }
+
+// GetValueString returns the value of a key in a map as a string.
+func GetValueString(data map[string]interface{}, key string) (string, error) {
```

Comment:
can we improve the name of the function?

Reply:
which one do you suggest ?